index.js

```diff
@@ -32,33 +23,29 @@ client.on("message", (message) => {
     message.reply(`Your access code is ${message.guild.id}`);
   } else if (message.content === `${prefix}reset`) {
     FirebaseHelpers.resetDB();
-  } else if (
-    message.content.substr(0, prefix.length + 8) === `${prefix}entries `
-  ) {
+  } else if (message.content.startsWith(`${prefix}entries`)) {
     const entriesFunction = async () => {
       let response = await FirebaseHelpers.numEntries(
         message.guild.name,
-        message.content.substr(prefix.length + 8)
+        message.content.substr(prefix.length + 7)
       );
-      message.reply(response);
+      message.reply(JSON.stringify(response));
     };
     entriesFunction();
```

Comment:
I get why this is done, but it's sloppy. Adjust the callback function to `on()` to be `async` so you can `await` this without having to do jank.